lib/docker-sync/dependencies/package_manager.rb

```diff
@@ -0,0 +1,18 @@
+module DockerSync
+  module Dependencies
+    module PackageManager
+      class << self
+        extend Forwardable
+        def_delegators :package_manager, :available?, :ensure!, :install_package
+      end
+
+      def self.package_manager
+        return @package_manager if defined? @package_manager
+        return @package_manager = PackageManager::Brew if PackageManager::Brew.available?
```

Comment:
Well, we **want** to end up with `PackageManager::None` actually: it will let the user use DockerSync in the case where all dependencies have been installed without brew (a user can have manually installed unison, fswatch, rsync or whatever dependency they need).